### PR TITLE
Add batch 7

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: snapshot.yaml
Comment (babenek) [inline snapshot.yaml:1174]: MIT License

**Review by babenek — APPROVED** [review]
Approved

**Review by babenek — DISMISSED** [review]
Approved

**Review by babenek — APPROVED** [review]
Approved

**Review by xDizzix — DISMISSED** [review]

**Review by babenek — APPROVED** [review]
Approved

**Review by babenek — DISMISSED** [review]
Approved

**Review by iuriimet — DISMISSED** [review]

**Review by babenek — APPROVED** [review]
Approved after merge conflict fix

**Review by iuriimet — APPROVED** [review]

**Pull request merged**
